scripts/prep-key.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+export DEPLOY_BRANCH=${DEPLOY_BRANCH:-master}
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/loklak_wok_android" -o "$TRAVIS_BRANCH" != "$DEPLOY_BRANCH" ]; then
```

Comment:
I guess, `and` operator shall be used instead

Reply:
@vibhcool It is fine in my opinion. `and` would mean only skip when all conditions are `true` i.e if it is a PR and repo is not loklak wok and branch is not master but we want to skip the deployment even when only one condition is true i.e skip when either it is a PR or it is not loklak wok repo or branch is not master. So, it should be `or` imo. `-o` act accordingly. It is correct.
Try this script for example 
```
#!/bin/bash
TRAVIS_PULL_REQUEST=false
TRAVIS_REPO_SLUG="fossasia/open-event-orga-server"
TRAVIS_BRANCH="master"
DEPLOY_BRANCH=${DEPLOY_BRANCH:-master}

if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-event-orga-server" -o  "$TRAVIS_BRANCH" != "$DEPLOY_BRANCH" ]; then
    echo "Skip production deployment for a very good reason."
    exit 0
fi
```